plane/source/docs/introduction.rst

```diff
@@ -52,7 +52,12 @@ Vehicle Configuration Types
    - SLT - fixed wing motor plus separate VTOL motors 
    - Tilt-Rotors - which tilt motors from vertical for VTOL to horizontal for fixed wing operation
    - Tailsitters - which takeoff and land with the nose vertical instead of horizontal
+- Rocket Gliders (Model Rocket boosted gliders configured as Elevon based flying wings or tailsitters)
 
+Model Rocket Telemetry and Event controller
```

Comment:
I dont think this section belongs here...as a separate plane configuration, it offers no guidance on use or setup....a separate plane configuration section on Rocket Gliders would be welcomed, however..until that is available adding the rocket glider line above should be on hold